src/main/java/Helper.java

```diff
@@ -0,0 +1,122 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.ArrayList;
+
+public class Helper extends Duke{
```

Comment:
Should there be whitespace in-between `Duke` and `{`?

Reply:
Yes, good spot!